connector/connect/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -111,8 +122,12 @@ class SparkConnectPlanner(plan: proto.Relation, session: SparkSession) {
     }
   }
 
-  private def transformUnresolvedExpression(exp: proto.Expression): UnresolvedAttribute = {
-    UnresolvedAttribute(exp.getUnresolvedAttribute.getPartsList.asScala.toSeq)
+  private def transformUnresolvedExpression(exp: proto.Expression): Expression = {
+    if (exp.getUnresolvedAttribute.getPartsCount == 1) {
+      Literal.create(exp.getUnresolvedAttribute.getParts(0), StringType)
```

Comment:
I need to change this is to match catalyst DSL. 

catalyst DSL generates a string literal for `a` in `relation.select("a")`. 

If we think catalyst DSL is the standard way to generate logical plan then we should also follow it in connect.